icon-font/social-logos.css

```diff
@@ -39,226 +32,261 @@
 /* Icons */
 
 
-.social-logo-amazon:before {
+.social-logo__amazon:before {
 	content:"\f600";
```

Comment:
this seems like a breaking change, maybe to also add the old rule, to not break existing usage?

Reply:
Yeah, that's a problem of https://github.com/Automattic/social-logos/pull/117, still trying to figure that out.